youtube_dl/extractor/adobepass.py

```diff
@@ -325,6 +320,11 @@
     'col080': {
         'name': 'Columbus Telephone'
     },
+    'Comcast_SSO': {
+        'name': 'Comcast XFINITY',
+        'username_field': 'user',
+        'password_field': 'passwd',
+    },
```

Comment:
It looked to me like it was mostly in alphabetical order by name already.  I thought it would make it easier to find entries when scrolling down the list.  If there's a different "method to the madness," let me know.  I'm sorry--I thought I was helping by organizing it.